motr/st/utils/motr_dgmode_io_st.sh

```diff
@@ -69,8 +69,8 @@ main()
 	dest_file="$MOTR_TEST_DIR/motr_dest"
 	rc=0
 
-	BLOCKSIZE=4096
-	BLOCKCOUNT=25
+	BLOCKSIZE=16384 #4096
+	BLOCKCOUNT=3
```

Comment:
[question]
We are having BLOCKCOUNT=3, specifically to ensure partial read/writes happens on 6 + 2 config, correct ?

Reply:
yes.